packages/upswyng-web/src/components/Search.tsx

```diff
@@ -10,12 +10,12 @@ const Search = () => {
   const searchQuery = getSearchParamVal(SEARCH_PARAM_QUERY);
   const { updateCurrentBannerColor } = React.useContext(BannerColorContext);
   updateCurrentBannerColor("");
-  const results = useSearchResults(searchQuery || "");
+  // const results = useSearchResults(searchQuery || "");
 
   return (
     <Container>
       <h1>Search</h1>
-      {searchQuery && <SearchResults results={results} />}
+      {/* {searchQuery && <SearchResults results={results} />} */}
```

Comment:
Now that `<ResourceList/>` expects a `TResource`, `<SearchResults/>` will need updated to accept the type returned from Algolia. To keep this PR from being more unruly than it already is, I'll tackle that in a separate one.

Reply:
Thank you.

Reply:
Addressed in #278